sw/device/tests/sim_dv/spi_tx_rx_test.c

```diff
@@ -231,95 +233,100 @@ static bool exp_irqs_fired(void) {
          fired_irqs[kDifSpiDeviceIrqGenericRxFull];
 }
 
+static void sync_testbench(void) {
+  // Set WFI status for testbench synchronization,
+  // no actual WFI instruction is issued.
+  test_status_set(kTestStatusInWfi);
+  test_status_set(kTestStatusInTest);
+}
```

Comment:
We usually use SW logger prints to let DV know when SW has reached a certain point in the test program, right? Why don't we use that here, too? Using the test status with a 'fake' WFI does not look right to me

Reply:
Okay, thanks for the clarification. Then I think this isn't blocking but I agree that a robust and uniform way to communicate such sync points between DV and SW is worth discussing and tracking in a separate issue

Reply:
Issue opened here https://github.com/lowRISC/opentitan/issues/17788

Reply:
Thanks!